crates/sui-graphql-client/src/lib.rs

```diff
@@ -965,239 +964,205 @@ mod tests {
 
     #[tokio::test]
     async fn test_balance_query() {
-        for (n, _) in NETWORKS.iter() {
-            let client = Client::new(n).unwrap();
-            let balance = client.balance("0x1".parse().unwrap(), None).await;
-            assert!(balance.is_ok(), "Balance query failed for network: {n}");
-        }
+        let client = Client::new_localhost();
```

Comment:
Can we maybe have a "get_test_client" method that is cfg(test) that reads from an Env Var for the url to use? This would give us the flexibility to run all of these network tests on every network we want (although we only care about doing it for this local env today)

Reply:
Yes, that sounds like a good idea! Thanks for the suggestion.